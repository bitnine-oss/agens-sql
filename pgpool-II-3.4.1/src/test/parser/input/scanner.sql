\pset standard_conforming_strings off
SELECT '\'';
SELECT '''';
SELECT 'A\nBC';
SELECT E'A\nBC';
SELECT U&'d\0061t\+000061'; --error
\pset standard_conforming_strings on
SELECT '\''; -- error
SELECT '''';
SELECT 'A\nBC';
SELECT E'A\nBC';

\pset server_encoding SQL_ASCII
# unicode ident
SELECT * FROM U&"d\0061t\+000061";
# unicode literal
SELECT E'd\u0061t\U00000061';
SELECT U&'d\0061t\+000061';
SELECT U&'d!0061t!+000061' UESCAPE '!';
SELECT E'\u3042\u3044\u3046\u3048\u304a'; -- error
SELECT U&'\3042\3044\3046\3048\304a'; -- error

\pset server_encoding UTF8
# unicode ident
SELECT * FROM U&"d\0061t\+000061";
# unicode literal
SELECT E'd\u0061t\U00000061';
SELECT U&'d\0061t\+000061';
SELECT U&'d!0061t!+000061' UESCAPE '!';
SELECT E'\u3042\u3044\u3046\u3048\u304a';
SELECT U&'\3042\3044\3046\3048\304a';
# surrogate pair
SELECT E'\ud800\udc00';
SELECT U&'\d800\dc00';
