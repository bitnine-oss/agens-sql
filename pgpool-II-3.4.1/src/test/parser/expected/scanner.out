pset 0
 SELECT ''''
 SELECT ''''
 SELECT 'A
BC'
 SELECT 'A
BC'
syntax error: SELECT U&'d\0061t\+000061'; --error
pset 1
syntax error: SELECT '\''; -- error
 SELECT ''''
 SELECT 'A\\nBC'
 SELECT 'A
BC'
pset 0
 SELECT * FROM "data"
 SELECT 'data'
 SELECT 'data'
 SELECT 'data'
syntax error: SELECT E'\u3042\u3044\u3046\u3048\u304a'; -- error
syntax error: SELECT U&'\3042\3044\3046\3048\304a'; -- error
pset 6
 SELECT * FROM "data"
 SELECT 'data'
 SELECT 'data'
 SELECT 'data'
 SELECT '„ÅÇ„ÅÑ„ÅÜ„Åà„Åä'
 SELECT '„ÅÇ„ÅÑ„ÅÜ„Åà„Åä'
 SELECT 'êÄÄ'
 SELECT 'êÄÄ'
